packages/92green-aws-rxjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@92green/92green-aws-rxjs",
-  "version": "0.11.1",
+  "version": "0.12.0-alpha",
```

Comment:
if this pr contains major changes should be bumped to 1.0.0 and please record any changes  made somewhere (CHANGELOG.md ?)

Reply:
Fairpoint on the changelog. Semver actually permits breaking changes on minor versions pre 1.0.0. Anything prior to 1.0.0 is considered unstable (or subject to change, probably not suitable for production), so I think we really should consider bumping it to 1.0.0 but I'm not sure if this is the place to make that call.

Reply:
I have added a CHANGELOG.md file detailing these changes and I think it is will worth discussing if we are ready for a 1.0.0 release.